### ISSUE TITLE
telescope integration not working in resident mode

### ISSUE THREAD

**Issue by vishal340** [RutaTang/quicknote.nvim#16]
is telescope integration only supposed to work on portable mode?

**Comment by RutaTang** [comment]
no,it is supposed to work with both of two modes. Might be a bug. I will see when I was out of busy. If anyone can fix it, I will very appreciate it!
